front/src/styles/scss/applications/operationalStudies/_simulationresults.scss

```diff
@@ -57,571 +43,4 @@ select.sm {
-  }
-
-  .route-aspect {
-    opacity: 0.5;
-    &.selected {
-      opacity: 1;
-      &.eco {
-        opacity: 1;
-      }
-    }
-  }
-
-  /* ALLOWANCES */
-  .mareco {
-    z-index: 2;
-  }
-  .allowance-line {
-    padding: 0.25rem;
-    border-radius: 4px;
-    transition: 0.2s;
-    &:hover {
-      background-color: var(--coolgray1);
-    }
-  }
-  .allowances-updating-loader {
-    position: absolute;
-    text-align: center;
-    background-color: rgba(255, 255, 255, 0.75);
-    width: calc(100% - 2rem);
-    height: 100%;
-    z-index: 3;
-    top: 0;
-    font-size: 4rem;
-  }
-
-  /* SPEEDSPACE */
-  .speedspacechart-container {
-    display: block;
-    position: relative;
-    width: 100%;
-    transition: 0.2s;
-    .showSettingsButton {
-      position: absolute;
-      border: none;
-      background: none;
-      cursor: pointer;
-      height: 100%;
-      color: var(--coolgray7);
-    }
-    .speedspace-chart {
-      display: flex;
-      background-color: white;
-      border-radius: 4px;
-      position: relative;
-      .showSettings {
-        overflow: hidden;
-        transition: 0.3s;
-      }
-      .btn-rotate {
-        position: absolute;
-        right: 2rem;
-        top: 1rem;
-      }
-      .area {
-        &.speed {
-          fill: rgba(48, 51, 131, 0.2);
-        }
-        &.slopes {
-          fill: var(--cyan);
-        }
-      }
-      .speed {
-        stroke: var(--blue);
-        stroke-width: 1;
-        &.pointer {
-          fill: var(--blue);
-          &.allowances {
-            fill: var(--orange);
-          }
-          &.eco {
-            fill: var(--green);
-          }
-        }
-        &.allowances {
-          stroke: var(--orange);
-        }
-        &.eco {
-          stroke: var(--green);
-        }
-        &.vmax {
-          stroke-width: 2;
-          stroke: var(--purple);
-        }
-        &.slopes {
-          stroke-width: 2;
-          stroke: var(--orange);
-        }
-        &.slopesHistogram {
-          stroke-width: 1;
-          stroke: var(--cyan);
-        }
-      }
-    }
-    .handle-tab-resize {
-      width: 100%;
-      text-align: center;
-      position: absolute;
-      bottom: -1.25rem;
-      font-size: 2rem;
-      transition: 0.2s;
-      color: var(--coolgray3);
-      &:hover {
-        color: var(--coolgray9);
-      }
-    }
-  }
-
-  /* Electrical profile legend */
-  .elecProf-modal {
-    position: absolute;
-    z-index: 0;
-    min-height: 5rem;
-    max-height: calc(100% - 5vh);
-    overflow: auto;
-    top: 0;
-    right: 8rem;
-    padding: 1rem;
-    background-color: #fff;
-    width: calc(25% - 5.5rem);
-    transition: 0.4s;
-    margin: 1rem;
-    border-radius: 4px;
-    border: 1px solid #d7d7d7;
-    @media screen and (min-width: 640px) {
-      max-width: 50%;
-    }
-
-    &-dark {
-      background-color: var(--coolgray13);
-      color: var(--coolgray1);
-      border-color: var(--coolgray13);
-    }
```

Comment:
we totally get rid off all dark mode config ?

Reply:
Hum I can ask the question to Nicolas :+1:

Reply:
Validated by Nicolas, we can remove the dark mode config :tada: